master/buildbot/process/buildstep.py

```diff
@@ -871,6 +872,9 @@ def startCommand(self, cmd, errorMessages=[]):
 
         # stdio is the first log
         self.stdio_log = stdio_log = self.addLog("stdio")
+        # enable timestamps if requested
+        self.stdio_log.setTimestampsMode(self.timestamp_stdio)
```

Comment:
The reason I used the name *timestamp_stdio* is because it only affects the *stdio* log.  The step may have other logs, and these will not be timestamped. I think the name is more descriptive then *prepend_timestamps*.